Docker/ods-api-web-sandbox-admin/alpine/mssql/run.sh

```diff
@@ -41,4 +41,17 @@ fi
 
 envsubst < /app/appsettings.template.json > /app/appsettings.json
 
+STATUS_ODS=1
+STATUS_ADMIN=1 
+until [[ $STATUS_ODS -eq 0 && $STATUS_ADMIN -eq 0 ]]; do
+  >&2 echo "ODS SQL Server is unavailable - sleeping"
+  /opt/mssql-tools18/bin/sqlcmd -W -h -1 -l 1 -U ${SQLSERVER_USER} -P "${SQLSERVER_PASSWORD}" -S $SQLSERVER_ODS_DATASOURCE -C || : > /dev/null 2>&1
+  STATUS_ODS=$?
+
+  >&2 echo "ADMIN SQL Server is unavailable - sleeping"
+  /opt/mssql-tools18/bin/sqlcmd -W -h -1 -l 1 -U ${SQLSERVER_USER} -P "${SQLSERVER_PASSWORD}" -S $SQLSERVER_ADMIN_DATASOURCE -C || : > /dev/null 2>&1
+  STATUS_ADMIN=$?
+  sleep 8
```

Comment:
Sometimes I get an error in Sandbox Admin telling that the Admin DB is in the middle of a restore. We might need to improve this check to account for that: 

![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/158314644/e984d11d-28d6-4274-87a3-ec0b6cae4137)

Reply:
Fixed by improving this check's logic to account for the DB being unavailable during a restore or upgrade.